src/compat/posix/include/unistd.h

```diff
@@ -17,6 +17,8 @@
 #include <sys/stat.h>
 #include <sys/cdefs.h>
 
+#include <sys/uio.h>
```

Comment:
I hope, malloc issue was fixed in the last commit:
https://github.com/embox/embox/pull/932/commits/3e4c0c35ad5d2b94e631073c93576e2ef6fbd818